src/kcas.ml

```diff
@@ -189,8 +190,12 @@ let cas loc before state =
 module Loc = struct
   type 'a t = 'a loc
 
-  let make after =
-    { state = Atomic.make @@ new_state after; id = Id.get_unique () }
+  let make ?(mode_val = Mode.obstruction_free) after =
```

Comment:
Maybe rename as `mode`?

Pretty much everything in OCaml is a value, so having `_val` as a suffix doesn't really add any information.

Reply:
Ok😅...will modify accordingly.